app/controllers/notes_controller.rb

```diff
@@ -1,12 +1,16 @@
 class NotesController < ApplicationController
-  before_action do
+  before_action except: :add_note do
```

Comment:
Please revert the changes in this file. (It's not related to the current grader permissions, and l'm not entirely sure why this change is necessary anyway. You could open up a separate pull request to discuss this change in more detail later.)